src/video/cocoa/SDL_cocoamessagebox.m

```diff
@@ -55,28 +55,18 @@ - (id)initWithParentWindow:(SDL_Window *)window
 - (void)showAlert:(NSAlert *)alert
 {
     if (nswindow) {
-#ifdef MAC_OS_X_VERSION_10_9
```

Comment:
I think removing this #ifdef will prevent SDL from being compiled on 10.8 and 10.7 systems. But I don't remember if those are meant to be supported compile targets right now, or just runtime targets. Maybe @slouken has an opinion - and maybe whether to keep the #ifdef depends on whether this will be backported to SDL2 or not.

Reply:
Accurate point. I couldn't find any reference to minimum versions for build or runtime. Given 10.8 (Mountain Lion) is 11 years old and unsupported as of 2015, this seemed a reasonable step forward.

Reply:
Ah, according to the readme SDL (2 and 3) only officially supports 10.9+ now https://github.com/libsdl-org/SDL/blob/main/docs/README-macos.md#command-line-build - so removing the `respondsToSelector:` check and the else codeblock should be good assuming the readme isn't wrong.

Reply:
I'll update the PR assuming we're in the right.